### PR TITLE
Migrate to BCrypt hashes

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java [hunk 1]
```diff
@@ -61,6 +73,19 @@ public void login_with_resource_owner_password_credentials_grant_should_provide_
         assertNotNull("The refresh token was null.", accessToken.getRefreshToken());
     }
 
+    @Test
+    @DatabaseSetup("/database_seed_login_with_old_hash.xml")
+    public void login_updates_to_bcrypt_hash() throws SQLException {
```
Comment (timopick) [inline src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java]: Codacy Issue found: [Method names should not contain underscores](https://www.codacy.com/app/t-pick/connector4java-integration-tests/file/2113725304/issues/source?bid=3145228&fileBranchId=3145235#l78)

Comment (timopick) [inline src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java]: Codacy Issue found: [Method names should not contain underscores](https://www.codacy.com/app/t-pick/connector4java-integration-tests/file/2113725304/issues/source?bid=3145228&fileBranchId=3145235#l78)

File: src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java [hunk 2]
```diff
@@ -61,6 +73,19 @@ public void login_with_resource_owner_password_credentials_grant_should_provide_
...
+        accessToken = OSIAM_CONNECTOR.retrieveAccessToken("marissa02", "koala", Scope.ADMIN);
+        String sql = "SELECT password FROM scim_user WHERE internal_id = 100005";
+        ResultSet rs = dataSource.getConnection().createStatement().executeQuery(sql);
```
Comment (timopick) [inline src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java]: Codacy Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://www.codacy.com/app/t-pick/connector4java-integration-tests/file/2113725304/issues/source?bid=3145228&fileBranchId=3145235#l81)

Comment (timopick) [inline src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java]: Codacy Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://www.codacy.com/app/t-pick/connector4java-integration-tests/file/2113725304/issues/source?bid=3145228&fileBranchId=3145235#l81)

**Comment by wallner** [comment]
Perfect. Thank you.

**Pull request merged**